root/static/scripts/guess-case/modes.js

```diff
@@ -282,3 +287,21 @@ exports.Sentence = assign({}, DefaultMode, {
     {url: {href: 'https://musicbrainz.org/doc/Style/Language', target: '_blank'}}
   ),
 });
+
+exports.Turkish = assign({}, DefaultMode, {
+  description: l(
+    'This mode capitalises almost all words of a sentence, the words ' +
+    'which are restricted to be capitalised are lowercased. Some words, often proper nouns, ' +
```

Comment:
1. Please remove `of a sentence` to prevent mix-up with _Sentence_ mode.
2. Moreover, `the words which are restricted to be capitalised are lowercased` is not very informative. Actually, just reuse the wording from _English_ mode, except that `connectives and question-suffixes` (taken from https://musicbrainz.org/doc/Style/Language/Turkish#In_English) should replace `articles and short prepositions`.
3. Finally, please mention that both `İ` and `ı` letters are handled.